lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -41,23 +45,50 @@ class Chef::Knife::VsphereVmSnapshot < Chef::Knife::BaseVsphereCommand
          :description => "Indicates whether to start the VM after a successful revert",
          :boolean => false
 
-  def run
+  option :memory,
+	 :long => "--memory",
+	 :description => "Indicates whether to dump the internal memory state in the snapshot",
+	 :boolean => false
+
+  def find_vm_in_folder(folder, vmname)
+    path = (folder.path[3..-1].map { |x| x[1] }.* '/')
```

Comment:
`path` doesn't seem to get used in this method. I think you could use [`traverse_folders_for_vm`](https://github.com/ezrapagel/knife-vsphere/blob/master/lib/chef/knife/base_vsphere_command.rb#L129-L140) to do the same thing.